FormElement.inc

```diff
@@ -59,6 +59,14 @@ class FormElement implements ArrayAccess {
   protected $index;
 
   /**
+   * Holds references to the original form default value for the form
+   * element.
+   *
+   * @var ReadOnlyProtected
```

Comment:
@DiegoPino

We did some digging and that alter is already being respected, no changes necessary.
The elements are instantiated immediately [after](https://github.com/Islandora/islandora_xml_forms/commit/b6f7ffbcdeeafad518fbe0e332804eca2efea638#diff-91737315d4cd0a424809819b2f889e7eR445) in [XMLForm](https://github.com/Islandora/islandora_xml_forms/blob/7.x/api/XMLForm.inc#L82).
Which initializes the elements and their controls.

Thanks!

----
William Panting
_Developer_
*[discoverygarden inc.](http://www.discoverygarden.ca) | Managing Digital Content*